napari/components/viewer_model.py

```diff
@@ -161,7 +161,7 @@ class ViewerModel(KeymapProvider, MousemapProvider, EventedModel):
         default_factory=EventedDict
     )
     # 2-tuple indicating height and width
-    _canvas_size: Tuple[int, int] = (600, 800)
+    _canvas_size: Tuple[int, int] = (800, 600)
```

Comment:
I'm confused. The docstring is not changed, but the order is reversed.

Reply:
This is because vispy convention and numpy convention are different but were both used on the napari side. I changed it so that we only deal with the numpy convention on the napari side. Clarified in docstring with [18d7964](https://github.com/napari/napari/pull/5432/commits/18d79643b152fa2f6d348abffd05996cef062577).